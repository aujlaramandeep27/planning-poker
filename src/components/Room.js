import React from 'react';
import { Link, Route, withRouter } from 'react-router-dom';
import Estimate from './Estimate';

export function Room({ match }) {
  const estimates = [
    1, 2, 3, 4, 5, 6, 7, 8, 9, 10
  ];
  return (
    <>
      <h1>Room {match.params.roomId}</h1>
      <Link to="/rooms/" className="uk-link-muted">
        <span data-uk-icon="home"/>
        <span className="uk-text-middle">Rooms</span>
      </Link>
      <Route path={match.path} render={() => <Estimate estimates={estimates}/>}/>
    </>
  )
}

export default withRouter(Room);
