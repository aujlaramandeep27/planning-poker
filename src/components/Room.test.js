import { Room } from './Room';

describe('Component: Room', () => {
  test('should render', () => {
    const match = { match: { params: { roomId: 'B' } } };
    expect(Room(match)).toMatchSnapshot();
  });
});
