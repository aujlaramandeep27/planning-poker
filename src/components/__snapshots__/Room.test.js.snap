// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Room should render 1`] = `
<React.Fragment>
  <h1>
    Room 
    B
  </h1>
  <RoomsLink />
  <Route
    render={[Function]}
  />
</React.Fragment>
`;
